lib/src/interface/local_platform.dart

```diff
@@ -48,5 +48,5 @@ class LocalPlatform extends Platform {
   String get version => io.Platform.version;
 
   @override
-  bool get ansiSupported => io.Platform.ansiSupported;
+  bool get ansiSupported => io.stdin.supportsAnsiEscapes;
```

Comment:
Hmmm.  On the other hand, this would break all clients.  I leave that call to you and @tvolkert. Happy either way.